### PR TITLE
framework/st_things: fix coding rule

### PR DESCRIPTION
**Pull request by KimDongEon**
Modification of coding rule violation

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/src/st_things/st_things_representation.c [hunk 1]
```diff
@@ -48,7 +48,7 @@ bool get_int_value(struct _st_things_representation *rep, const char *key, int64
 	RET_FALSE_IF_PARAM_IS_NULL(key);
 	RET_FALSE_IF_PARAM_IS_NULL(value);
 
-	return OCRepPayloadGetPropInt((OCRepPayload *)rep->payload, key, value);
+	return OCRepPayloadGetPropInt((OCRepPayload *) rep->payload, key, value);
```
Comment (sunghan-chang) [inline framework/src/st_things/st_things_representation.c]: revert

File: framework/src/st_things/st_things_representation.c [hunk 2]
```diff
@@ -70,7 +70,7 @@ bool get_byte_value(struct _st_things_representation *rep, const char *key, uint
 	RET_FALSE_IF_PARAM_IS_NULL(size);
 
 	OCByteString byte_value = { NULL, 0 };
-	if (!OCRepPayloadGetPropByteString((OCRepPayload *)rep->payload, key, &byte_value)) {
+	if (!OCRepPayloadGetPropByteString((OCRepPayload *) rep->payload, key, &byte_value)) {
```
Comment (sunghan-chang) [inline framework/src/st_things/st_things_representation.c]: revert

File: framework/src/st_things/st_things_request_handler.c [hunk 3]
```diff
@@ -480,161 +480,151 @@ static bool add_property_in_post_req_msg(st_things_set_request_message_s *req_ms
...
+				ST_LOG_V(ST_ERROR, "Failed to set the object value of '%s' in request message", prop->key);
+				OCRepPayloadDestroy(value);
 			}
+		} else {
+			ST_LOG_V(ST_ERROR, "Failed to get the object value of '%s' for request message", prop->key);
+		}
+	}
+	break;
+	case BYTE_ID: {
+		OCByteString *byte_value = (OCByteString *) util_calloc(1, sizeof(OCByteString));
```
Comment (jeongchanKim) [inline framework/src/st_things/st_things_request_handler.c]: remove space after type casting

**Comment by sunghan-chang** [comment]
@KimDongEon Any update?

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by KimDongEon** [comment]
@jeongchanKim 
modify

**Pull request merged**